app/frontend/actions.ts

```diff
@@ -58,11 +58,7 @@ export default (store: Store) => {
   const {setState, getState} = store
 
   const setAuthMethod = (state: State, option: AuthMethodType): void => {
-    setState({
-      authMethod: option,
-      shouldShowExportOption:
-        option === AuthMethodType.MNEMONIC || option === AuthMethodType.KEY_FILE,
-    })
+    setState({authMethod: option})
```

Comment:
I would rename the `option` argument to `authMethod` but I guess that's a bit unrelated to this refactor....